team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -78,11 +80,20 @@ private TreeNode getBestMove(TreeNode root) {
 		TreeNode best = null;
 
 		for (TreeNode node : root.children) {
-			int wins = node.getWins();
+			/*
+			 * Flip the wins, we need opponent wins due to how the pattern of our tree works:
+			 * When the board is passed to us, the root will be OUR turn
+			 * the children will all be of the opponents turn
+			 * so in a sense we actually want the node that is ~losing~ the most
+			 */
+
+			int wins = node.getVisits() - node.getWins();
```

Comment:
- @JadenBalogh see above my removal of line 50, not sure if you remember this conversation, but it turns out we do want to pass the root as our color and then leave the children node as the opposing turn, easier to explain over voice which I can do Sunday if you want, but it's essentially because the edge is the action itself and then the node corresponds to the game state so the edge is our action, and then the node will be the opposing teams game state to start with, so we essentially find how many losses for the opposing player